test/format_strftime_test.exs

```diff
@@ -17,6 +17,20 @@ defmodule DateFormatTest.FormatStrftime do
     end)
   end
 
+  test "success!" do
+    date = Timex.to_datetime({2013,03,02})
+
+    formatter = Timex.Format.DateTime.Formatters.Strftime
```

Comment:
You don't need this, just use the `format/2` local which is defined in this module, like the other tests

Reply:
Actually why is this test added? I think it can be removed entirely, but I'm open to being swayed if you think it is necessary

Reply:
I just ran the coverage report and saw that this module was not hitting all lines, so I thought I'd just add two tests to cover it completely. So you're right in that it does not affect what I added, but I thought that I should raise the coverage instead of decrease it :)